delphi/translators/for2py/genPGM.py

```diff
@@ -279,18 +306,18 @@ def genPgm(self, node, state, fnNames, call_source):
             id_specList = self.make_identifier_spec(
                 loopName, indexName, {}, state
             )
-
             loopCall = {
                 "name": loopName,
                 "input": [
                     {
-                      "variable": variable,
-                      "index": -1,
+                        "name": variable,
```

Comment:
@pauldhein Looks like the `variable` key has been changed back to `name`, so this will require a couple of changes in `GroundedFunctionNetwork`.

Reply:
Noted and updated.